### DISCUSSION TITLE
OpenRV file size

### DISCUSSION THREAD

**Discussion by rubikstriangle** [AcademySoftwareFoundation/OpenRV#62] [Build Issues] [answered]
After building OpenRV and creating an install the size of that folder is 5.8GB. Autodesk RV is a bit over 2GB.  Does the size of my OpenRV install sound about right to you guys or did I do something wrong?

**Comment by geffrak** [comment, marked as answer]
On OSX, OpenRV (1.29GB) is smaller than RV(1.7GB). What OS are you using? Are you building Debug or Release? In debug, we are "installing" the pdb/.dsym files, some of which are really big. One thing that de don't do in OpenRV that we are used to do in RV is stripping the Linux binaries.

**Reply by rubikstriangle** [reply]
That's probably it.  I followed config_linux_centos7.md verbatim and I don't think it specified  a build type, does it default to debug?  I'll try again with -DCMAKE_BUILD_TYPE=Release to see that that gives me.

**Reply by rubikstriangle** [reply]
I built openRv again, this time explicitly setting the release flag and the file size came out to 1.3GB.  Thanks for the answer!
